README.md

```diff
@@ -211,34 +221,69 @@ Link to other services:
 docker-compose -f docker-compose.yaml -f docker-compose.link.yaml up
```

Comment:
I'd maybe make it clearer that the `scripts/start` script referenced above is a way of avoiding all of these steps?

Reply:
Do we need these extra steps given we have the helper script?